setupbase.py

```diff
@@ -1,735 +0,0 @@
-
-    Splits path recursively with os.path.split().
-    """
-    (head, tail) = os.path.split(path)
-    if not head or (not tail and head == path):
-        if head:
-            yield head
-        if tail or not head:
-            yield tail
-        return
-    for p in _iexplode_path(head):
-        yield p
-    yield tail
-
-
-def _translate_glob(pat):
-    """Translate a glob PATTERN to a regular expression."""
-    translated_parts = []
-    for part in _iexplode_path(pat):
-        translated_parts.append(_translate_glob_part(part))
-    os_sep_class = '[%s]' % re.escape(SEPARATORS)
-    res = _join_translated(translated_parts, os_sep_class)
-    return '{res}\\Z(?ms)'.format(res=res)
-
-
-def _join_translated(translated_parts, os_sep_class):
-    """Join translated glob pattern parts.
-
-    This is different from a simple join, as care need to be taken
-    to allow ** to match ZERO or more directories.
-    """
-    res = ''
-    for part in translated_parts[:-1]:
-        if part == '.*':
-            # drop separator, since it is optional
-            # (** matches ZERO or more dirs)
-            res += part
-        else:
-            res += part + os_sep_class
-
-    if translated_parts[-1] == '.*':
-        # Final part is **
-        res += '.+'
-        # Follow stdlib/git convention of matching all sub files/directories:
-        res += '({os_sep_class}?.*)?'.format(os_sep_class=os_sep_class)
-    else:
-        res += translated_parts[-1]
-    return res
-
-
-def _translate_glob_part(pat):
-    """Translate a glob PATTERN PART to a regular expression."""
-    # Code modified from Python 3 standard lib fnmatch:
-    if pat == '**':
-        return '.*'
-    i, n = 0, len(pat)
-    res = []
-    while i < n:
-        c = pat[i]
-        i = i + 1
-        if c == '*':
-            # Match anything but path separators:
-            res.append('[^%s]*' % SEPARATORS)
-        elif c == '?':
-            res.append('[^%s]?' % SEPARATORS)
-        elif c == '[':
-            j = i
-            if j < n and pat[j] == '!':
-                j = j + 1
-            if j < n and pat[j] == ']':
-                j = j + 1
-            while j < n and pat[j] != ']':
-                j = j + 1
-            if j >= n:
-                res.append('\\[')
-            else:
-                stuff = pat[i:j].replace('\\', '\\\\')
-                i = j + 1
-                if stuff[0] == '!':
-                    stuff = '^' + stuff[1:]
-                elif stuff[0] == '^':
-                    stuff = '\\' + stuff
-                res.append('[%s]' % stuff)
-        else:
-            res.append(re.escape(c))
-    return ''.join(res)
-
-
-
-# pythreejs specific functions to handle autogen:
-
-# Call this after running wrapper command, but before running wrapped command:
-
-def update_packages(command):
-    """update build_py options to get packages changes"""
-    command.distribution.packages = find_packages()
```

Comment:
This bit is critical!

Reply:
This one I actually missed, thanks for the catch. Will fix